compiler/rustc_mir_build/src/thir/pattern/_match.rs

```diff
@@ -861,6 +814,20 @@ impl<'tcx> Constructor<'tcx> {
         }
     }
 
+    fn as_intrange(&self) -> Option<&IntRange<'tcx>> {
```

Comment:
I would prefer `as_int_range` here, as it's more readable (and follows typical naming conventions).